src/test/java/com/google/collegeplanner/PlannerTest.java

```diff
@@ -37,9 +37,50 @@
 @RunWith(JUnit4.class)
 public final class PlannerTest {
```

Comment:
missing failure test cases:

 - what happens when no schedule can be made due to conflicts / not enough credits or semesters
 - what happens when no schedule can be made due to an infinite prereq loop
 - what happens when you have an empty input? or invalid input?
 - ...and any other unique failure mode you can think of